### PR TITLE
chore: add skeleton files for MDC-based implementation of chips

### PR DESCRIPTION
**Pull request by vanessanschmitt**

### CODE REVIEW

File: src/material-experimental/mdc-chips/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+sass_binary(
+  name = "chips_scss",
+  src = "chips.scss",
+  include_paths = [
+    "external/npm/node_modules",
+  ],
+  deps = [
+    ":chips_scss_lib",
```
Comment (mmalerba) [inline src/material-experimental/mdc-chips/BUILD.bazel]: I think this dep can be removed (my bad for adding it in the original skeleton PR)

File: src/material-experimental/mdc-chips/_mdc-chips.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@import '../mdc-helpers/mdc-helpers';
+
+@mixin mat-chips-theme-mdc($theme) {
+  @include mat-using-mdc-theme($theme) {
+    // TODO: MDC theme styles here.
+  }
+}
+
+@mixin mat-chips-typography-mdc($config) {
+  @if config {
```
Comment (mmalerba) [inline src/material-experimental/mdc-chips/_mdc-chips.scss]: I moved the `@if` logic into the mixin, you can just do this now:

```sass
@include mat-using-mdc-typography($config) {
  // TODO: MDC typography styles here.
}
```

File: src/material-experimental/mdc-chips/chip.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/**
```
Comment (jelbourn) [inline src/material-experimental/mdc-chips/chip.ts]: I would name this file `chip-cell.ts` since `MatChip` by itself with no role should still be a thing.

**Comment by vanessanschmitt** [comment]
Thanks for the feedback! I pushed the requested changes.

**Pull request closed without merging**